recipe/build_base.sh

```diff
@@ -1,22 +1,12 @@
 #!/bin/bash
 
-cat <<EOF > conda_mpl_config.cfg
-[directories]
-basedirlist = $PREFIX
+set -ex
 
-[packages]
-tests = False
-toolkit_tests = False
-sample_data = False
+export AR=$GCC_AR
 
-[libs]
-system_freetype = True
+export MESON_ARGS="${MESON_ARGS} --buildtype=release --prefix=${PREFIX} -Dlibdir=lib --pkg-config-path=${PREFIX}/lib/pkgconfig -Dsystem-freetype=true -Dsystem-qhull=true"
 
-EOF
-
-cat conda_mpl_config.cfg
-sed -i.bak "s|/usr/local|${PREFIX}|" setupext.py
-
-export MPLSETUPCFG=conda_mpl_config.cfg
-
-$PYTHON -m pip install --no-deps --no-build-isolation -vv .
+mkdir builddir
+$PYTHON -m mesonbuild.mesonmain setup builddir $MESON_ARGS
+$PYTHON -m build --wheel --no-isolation --skip-dependency-check -Cbuilddir=builddir
```

Comment:
I'm not sure the extra `builddir` is necessary; just need to pass all of `$MESON_ARGS` through all the layers. I think that should just mean prefixing every one with `-Csetup-args=`.

Reply:
Yeah I tried dropping it at one point and saw some errors (particularly [with macOS ARM]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=963536&view=logs&j=8ca717d7-2f91-593f-2e86-7b14f42b0119&t=f56d028c-9d05-5be8-63d1-db69e7fd673f&l=1275 )). So put it back

We can certainly try again. Though would be nice to get to a point where all the builds are passing before doing clean up (so we can return to a working state more easily)